airflow/bin/cli.py

```diff
@@ -1782,6 +1783,13 @@ class CLIFactory(object):
                 "all the failed tasks for the backfill date range "
                 "instead of throwing exceptions"),
             "store_true"),
+        'run_backwards': Arg(
+            ("-B", "--run_backwards",),
+            (
+                "if set, the backfill will run tasks from the most "
+                "recent day first "
+                "instead of throwing exceptions"),
```

Comment:
How about adding one more line "this option will fail if the DAG is depends_on_past."?

Reply:
done